configuration/scripts/options/set_nml.cmip

```diff
@@ -0,0 +1,143 @@
+    histfreq           = 'm','d','n','x','x'
+    histfreq_n         =  1,1,15,1,1
+    hist_avg           = .true.,.true.,.false.,.false,.false.
+    f_aice             = 'md'
+    f_aice_init        = 'md'
+    f_aicen            = 'md'
+    f_siage            = 'md'
+    f_siconc           = 'md'
+    f_sifb             = 'md'
+    f_siflcondbot      = 'md'
+    f_siflcondtop      = 'md'
+    f_sifllattop       = 'md'
+    f_sifllwdtop       = 'md'
+    f_sifllwutop       = 'md'
+    f_siflsensbot      = 'md'
+    f_siflsenstop      = 'md'
+    f_siflswdtop       = 'md'
+    f_siflswutop       = 'md'
+    f_sihc             = 'md'
+    f_siitdconc        = 'md'
+    f_siitdsnconc      = 'md'
+    f_siitdsnthick     = 'md'
+    f_siitdthick       = 'md'
+    f_simpconc         = 'md'
+    f_simpeffconc      = 'md'
+    f_simprefrozen     = 'md'
+    f_simpthick        = 'md'
+    f_sirdgconc        = 'md'
+    f_sirdgthick       = 'md'
+    f_sisali           = 'md'
+    f_sisaltmass       = 'md'
+    f_sisnhc           = 'md'
+    f_sisnthick        = 'md'
+    f_sitempbot        = 'md'
+    f_sitempsnic       = 'md'
+    f_sitemptop        = 'md'
+    f_sithick          = 'md'
+    f_sitimefrac       = 'md'
+    f_siu              = 'md'
+    f_siv              = 'md' 
+    f_sicompstren      = 'm'
+    f_sidconcdyn       = 'm'
+    f_sidconcth        = 'm'
+    f_sidivvel         = 'n'
+    f_sidmassdyn       = 'm'
+    f_sidmassevapsubl  = 'm'
+    f_sidmassgrowthbot = 'm'
+    f_sidmassgrowthsi  = 'm'
+    f_sidmassgrowthwat = 'm'
+    f_sidmassmeltbot   = 'm'
+    f_sidmassmeltlat   = 'm'
+    f_sidmassmelttop   = 'm'
+    f_sidmassth        = 'm'
+    f_sidmasstranx     = 'm'
+    f_sidmasstrany     = 'm'
+    f_sidragbot        = 'm'
+    f_sidragtop        = 'm'
+    f_siflfwbot        = 'm'
+    f_siflfwdrain      = 'm'
+    f_siflsaltbot      = 'm'
+    f_siforcecoriolx   = 'm'
+    f_siforcecorioly   = 'm'
+    f_siforceintstrx   = 'm'
+    f_siforceintstry   = 'm'
+    f_siforcetiltx     = 'm'
+    f_siforcetilty     = 'm'
+    f_simass           = 'm'
+    f_sipr             = 'm'
+    f_sishearvel       = 'n'
+    f_sisnconc         = 'm'
+    f_sisndmassdyn     = 'm'
+    f_sisndmassmelt    = 'm'
+    f_sisndmasssi      = 'm'
+    f_sisndmasssnf     = 'm'
+    f_sisndmasssubl    = 'm'
+    f_sisndmasswind    = 'm'
+    f_sisnmass         = 'm'
+    f_sispeed          = 'm'
+    f_sistressave      = 'n'
+    f_sistressmax      = 'n'
```

Comment:
So 'n' means instantaneous?  Is that documented?  We used to just have those fields hardwired because averages are usually not computed correctly and can be very misleading.